install-dependencies.sh

```diff
@@ -4,7 +4,7 @@ git clone -b hotfix/0.4 https://github.com/ebivariation/biodata.git
 git clone -b hotfix/0.5 https://github.com/ebivariation/opencga.git
 git clone -b develop https://github.com/ebivariation/variation-commons.git
 
-cd biodata && mvn install
+cd biodata && git checkout dbfeee0fd551dfb36ac40a2e70d612cf7439f6db && mvn install
```

Comment:
This is irrelevant now- I've rebased on to develop where this install script isn't needed due to the artifactory.